Vsxmd/Vsxmd.md

```diff
@@ -529,6 +546,23 @@ The element at the specified position in the `source` sequence.
 | ---- | ----------- |
 | TSource | The type of the elements of `source`. |
 
+<a name='M-Vsxmd-Units-Extensions-ParseBrackets-System-String-'></a>
+### ParseBrackets(str) `method`
+
+##### Summary
+
+Parse the brackets from XML-safe to Markdown formatting.
+
+##### Returns
+
+String with converted brackets
```

Comment:
The build is failed around here. It looks like you forget another round of build to generate the markdown file.